c2_utils/src/mfx_va_frame_pool_allocator.cpp

```diff
@@ -41,15 +47,25 @@ mfxStatus MfxVaFramePoolAllocator::AllocFrames(mfxFrameAllocRequest *request,
     std::lock_guard<std::mutex> lock(m_mutex);
     // The max buffer needed is calculated from c2 framework (CCodecBufferChannel.cpp):
     // output->maxDequeueBuffers = numOutputSlots + reorderDepth.value(0) + kRenderingDepth(3);
-    // if (!secure) {
-    //      output->maxDequeueBuffers += numInputSlots;
-    // }
-    // numInputSlots = inputDelayValue(input_delay_) + pipelineDelayValue(0) + kSmoothnessFactor(4);
     // numOutputSlots = outputDelayValue(output_delay_) + kSmoothnessFactor(4);
-    // buffers_count = output_delay_ + kSmoothnessFactor(4) + kRenderingDepth(3) + input_delay_(2) + kSmoothnessFactor(4)
-    int max_buffers = m_uSuggestBufferCnt + 13;
+    // buffers_count = output_delay_ + kSmoothnessFactor(4) + kRenderingDepth(3)
```

Comment:
which counter is bigger? the original m_uSuggestBufferCnt + 13, or output_delay_ + kSmoothnessFactor(4) + kRenderingDepth(3) ?

Reply:
The original m_uSuggestBufferCnt + 13 is from Android R. This is bigger. But on Android S, it is updated from m_uSuggestBufferCnt + 13 to (output_delay_ + kSmoothnessFactor(4) + kRenderingDepth(3)).

Reply:
Ok.